src/aiotk/testing.py

```diff
@@ -24,87 +39,106 @@ def call_with_minimal_args(f, **kwds):
     return f(**kwds)
 
 
-class OutputStreamAdapter(object):
+class OutputStreamAdapter(StreamWriter):
     """StreamWriter that wraps a file-like object."""
 
-    def __init__(self, stream):
+    def __init__(self, stream: TextIO) -> None:
         self._stream = stream
 
-    def write(self, data):
-        assert isinstance(data, bytes)
+    def write(self, data: bytes) -> None:
         self._stream.write(data.decode('utf-8'))
 
-    def writelines(self, lines):
+    def writelines(self, lines: Iterable[bytes]) -> None:
         for line in lines:
             self._stream.write(line.decode('utf-8'))
             self._stream.write('\n')
 
-    def can_write_eof(self):
+    def can_write_eof(self) -> bool:
         return True
 
     # TODO: test this without closing the "real" sys.stdout!
-    def write_eof(self):
+    def write_eof(self) -> None:
         # self._stream.close()
         pass
 
+    # NOTE: impossible to please `mypy` because documented signature is
+    #       incomplete.
     async def drain(self):
```

Comment:
**Note to self**: report this feedback to `mypy` devs to ensure they are aware of this limitation in `mypy`.